README.md

```diff
@@ -8,3 +8,30 @@ the [Secret Store CSI
 Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver). Allows you
 to access secrets stored in Secret Manager as files mounted in Kubernetes pods.
 
+## Build and deploy notes
+
+**WARNING:** These are preliminary notes to aid in development, this plugin is currently not functional.
+
+* Create a new GKE cluster with K8S 1.16+
+* Install [Secret Store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) to the cluster.
+```shell
+$ kubectl apply -f deploy/rbac-secretproviderclass.yaml
+$ kubectl apply -f deploy/csidriver.yaml
+$ kubectl apply -f deploy/secrets-store.csi.x-k8s.io_secretproviderclasses.yaml
+$ kubectl apply -f deploy/secrets-store-csi-driver.yaml
+```
+* Use [Google Cloud Build](https://cloud.google.com/run/docs/building/containers#building_using) and [Container Registry](https://cloud.google.com/container-registry/docs/quickstart) to build and host the plugin docker image.
```

Comment:
remove

Reply:
Are you suggesting I remove the entire bullet? My intent is to give other developers a heads up as to what the script does since it could require the user to enable APIs or incur unexpected fees. Part of the roadmap will be to build/distribute an image so that users in the future can just apply some yaml without these scripts.